plugins/node/opentelemetry-instrumentation-azure-functions/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenTelemetry Azure Functions Instrumentation for Node.js
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for the [`@azure/functions`](https://learn.microsoft.com/azure/azure-functions/functions-reference-node?pivots=nodejs-model-v4) module, which may be loaded using the [`@opentelemetry/sdk-trace-node`](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-sdk-trace-node) package and is included in the [`@opentelemetry/auto-instrumentations-node`](https://www.npmjs.com/package/@opentelemetry/auto-instrumentations-node) bundle.
+
+If total installation size is not constrained, it is recommended to use the [`@opentelemetry/auto-instrumentations-node`](https://www.npmjs.com/package/@opentelemetry/auto-instrumentations-node) bundle with [@opentelemetry/sdk-node](`https://www.npmjs.com/package/@opentelemetry/sdk-node`) for the most seamless instrumentation experience.
```

Comment:
This package is not being added as a dependency to auto-instrumentations-node project, maybe add the dependency in the project or remove the documentation and add it at a later time when available.

Reply:
Yes, we would like it added to the auto-instrumentations package. Not sure on the ordering if I can add it as a dependency in this PR or would need to wait until after the first release

Reply:
I think it could be added as part of this change, if you like. Add a dependency to auto-instrumentations-node on version 0.1.0 of this package.  I've not seen a new instrumentation added to auto-instrumentations-node in the first commit before, but then they don't get added that frequently.

If it helps, here is a diff showing the last instr added to auto-instrumentations-node: https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2131/files

Reply:
Removed readme note for now. I'll add to auto package separately in the future